Formula/lunarvim.rb

```diff
@@ -0,0 +1,35 @@
+class Lunarvim < Formula
+  desc "IDE layer for Neovim with sane defaults"
+  homepage "https://www.lunarvim.org"
+  url "https://github.com/LunarVim/LunarVim/archive/refs/tags/1.1.4.tar.gz"
+  sha256 "3ff64f075685565225b3d07b33844c75d0fda1e8682c822384997fecaedbe273"
+  license "GPL-3.0-or-later"
+
+  depends_on "neovim"
+  depends_on "node"
+  depends_on "python@3.10"
+  depends_on "rust"
+
+  def install
+    inreplace "#{buildpath}/utils/bin/lvim.template" do |s|
+      s.gsub!(/RUNTIME_DIR_VAR/, "\"#{share}\"")
+      s.gsub!(/CONFIG_DIR_VAR/, "\"$HOME/.config/lvim\"")
+      s.gsub!(/CACHE_DIR_VAR/, "\"$HOME/.cache/lvim\"")
```

Comment:
Why not within Homebrew?

Reply:
As in why not use the formula's prefix as the config and/or cache locations? I threw this together as a first-pass to simply mimic the existing installation procedure (see [https://github.com/LunarVim/LunarVim/blob/rolling/utils/installer/install.sh#L9-L16]); these locations reflect that. 

I have a local branch where I'm looking at at least having a default configuration within the installation directory (in `pkgetc`, perhaps?). That said, is there a convention for user-specific runtime configuration for a Homebrew-managed package?

Reply:
I'm not sure this will work, since the installer actually needs to set up both the config and runtime directories. 

related: https://github.com/orgs/Homebrew/discussions/778